scripts/helmcharts/openreplay/templates/job.yaml

```diff
@@ -131,6 +135,7 @@ spec:
           mountPath: /opt/migrations/
       {{- end}}
       {{- if .Values.global.enterpriseEditionLicense }}
+      {{- if .Values.clickhouse.enabled }}
```

Comment:
Clickhouse is a mandatory component for EE edition. So we don't need the else condition here. The ee check will be done by the License check in the above line.

Reply:
Ah didn't realize how mandatory this was for the EE edition. Should be resolved now with https://github.com/openreplay/openreplay/pull/757/commits/dd147fae9bdc428e542f9d85e9b8681c330b5c75